src/index.css

```diff
@@ -11,6 +11,7 @@
   --note-border-dark-clr: #555555;
   --background: var(--light-bg-clr);
   --text-primary: var(--font-clr);
+  --dark-purple-clr: #4c3a51;
```

Comment:
You can be consistent with the namings either user var or hex. It will look good